modules/local/report_aggregate.nf

```diff
@@ -12,12 +12,16 @@ process REPORT_SUMMARIES{
 
     output:
     path("final_report.tsv"), emit: final_report
-    path("*_flattened.json"), emit: flattened_files
+    path("final_report_transposed.tsv"), emit: final_report_transposed
+    path("final_report_flattened.json"), emit: flattened_files
+    path("*${sample_flat_suffix}"), emit: flat_samples
+    val sample_flat_suffix, emit: sample_suffix
     path "versions.yml", emit: versions
 
     script:
+    sample_flat_suffix = "_flat_sample.json"
```

Comment:
Good call, I will add this to the nextflow.config. I think I got lazy in defining it and I was trying to avoid updating the nextflow_schema.json. I will make the change!

Reply:
fixed in: 44abe9249cc40e86b9ed1b5179e270934e4b1a13